HighwayPatrol.h

```diff
@@ -2,12 +2,16 @@
 
 #include "Vehicle.h"
 
-
 struct Highway;
 
 struct HighwayPatrol : Vehicle 
 {
     HighwayPatrol();
+    
+    ~HighwayPatrol() override;
+    HighwayPatrol(const HighwayPatrol&) = default;
+    HighwayPatrol& operator=(const HighwayPatrol&) = default;
+    
```

Comment:
These should be defined outside of the class.  It's not giving you a warning because the destructor is currently defined outside of the class, which suppresses the warning.
- [ ] revise